NebulaPatcher/Patches/Transpilers/BuildTool_Common_Transpiler .cs

```diff
@@ -0,0 +1,34 @@
+﻿using HarmonyLib;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace NebulaPatcher.Patches.Transpiler
+{
+    class BuildTool_Common_Transpiler
+    {
+        [HarmonyTranspiler]
+        [HarmonyPatch(typeof(BuildTool_Click), nameof(BuildTool_Click.CreatePrebuilds))]
+        [HarmonyPatch(typeof(BuildTool_Path), nameof(BuildTool_Path.CreatePrebuilds))]
+        [HarmonyPatch(typeof(BuildTool_Inserter), nameof(BuildTool_Inserter.CreatePrebuilds))]
+        static IEnumerable<CodeInstruction> CreatePrebuilds_Transpiler(IEnumerable<CodeInstruction> instructions)
```

Comment:
Do we need to check we are in an MP game here?

Reply:
So in 0.6.x this was how the function worked already, it assumed flag was 1 and then the code after it would set it to 0 if the player didn't have the required items. In 0.7.x they don't set it to 1 there, but the code after should still set it to 0 if the player doesn't have the required items so it should be fine in SP as well AFAIK